testgrid/config.yaml

```diff
@@ -6566,6 +6568,11 @@ dashboards:
     test_group_name: issue-creator
     code_search_url_template:
       url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
+  - name: push-prow
+    description: pushes prow on each commit
```

Comment:
can we be a little more descriptive about what "push" means in this context? what does `prow/bump.sh` actually do?

Reply:
I would also be fine with something like "runs prow/bump.sh on each commit" and then documenting what that script does in its header or something.  Right now it's... some shell and some bazel

Reply:
Done

Reply:
Late lgtm. Thanks